### ISSUE TITLE
More Discussion on Pointcloud Optimization

### ISSUE THREAD

**Issue by ZhenyuSun-Walker** [TQTQliu/MVSGaussian#53]
Dear Sir. Now I can get a nicer pointcloud from the 4 target-view images. However, when I do the gaussian optimization stage, although now it can properly read the ply file and do the optimization, but the rendering result after the optimization is quite bad, and the input .ply almost does not change after the optimization.

The better pointcloud form the 4 target view images as the initialization point cloud is shown as below:
![image](https://github.com/user-attachments/assets/7fe07b4e-4657-4472-b3e6-cdf6c3b1fcc1)

The gaussian render images of the target view is blank：
![image](https://github.com/user-attachments/assets/26039877-8888-4895-b292-5782494472fb)

Anticipating for your reply about such phenomenon.

**Comment by ZhenyuSun-Walker** [comment]
Actually I can also see nothing

**Comment by TQTQliu** [comment]
This is so weird.
Please check that `scale_factor` [here](https://github.com/TQTQliu/MVSGaussian/blob/b9a0fdd822dffad73cdcefb5b2120865c1064109/lib/networks/mvsgs/network.py#L146-L159) is the same as the one in config file.

**Comment by ZhenyuSun-Walker** [comment]
Yes, the scale_factor here is the same as the one in config file. 
Actually, the better pointcloud before the gaussian optimization is generated by using other strategies, but is from the 4 target view-image rendered from the net work, so I think it does not matter with the network part. However, the depth map directly from the network is not good as the one on the example datasets, and I think it contributes to the poor pointcloud initilization, thus I try the better strategy.

**Comment by TQTQliu** [comment]
Hi, you mean the depth map was obtained by another method? And then run `fusion.py` we provided to get the initialization point cloud ? If so, I suspect that the scale of the extrinsics is not the same as the scale of the depth, you can try setting `scale_factor=1` in the config file, and then run `fusion.py` and train 3DGS.

**Comment by ZhenyuSun-Walker** [comment]
Not really. The better pointcloud is generate based on th 4 taret-view images, and it does not rely on the depth map.
BTW, if I simply change the depth map of the 4 target-view images, then when I run fusion.py, it can also work normally and may produce better pointcloud as well?

**Comment by ZhenyuSun-Walker** [comment]
Sir, how can I get the corresponding camera intrinsic of the 4 target-view images?
